tests/manifests/import-vm-v2v-apb.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: v1
+kind: Template
+metadata:
+  name: v2v-template
+  annotations:
+    description: "Template for v2v import from VMWare"
+objects:
+- apiVersion: servicecatalog.k8s.io/v1beta1
+  kind: ServiceInstance
+  metadata:
+    name: import-vm-apb
+  spec:
+    clusterServiceClassExternalName: ${APB_NAME}
+    clusterServicePlanExternalName: vmware
+    parameters:
+      admin_user: ${OCP_ID}
+      admin_password: ${OCP_PASS}
+      url: ${VMWARE_URL}
+      vm_name: ${VM_NAME}
+      username: ${VMWARE_ID}
+      password: ${VMWARE_PASS}
```

Comment:
What about `os_type`? Do we want to have it here?

Reply:
included

Reply:
but it will only be useful when testing windows VMs

Reply:
correct